apps/condo/pages/settings/contactRole/[id]/index.tsx

```diff
@@ -70,6 +73,20 @@ const TheContactRolePage = (): JSX.Element => {
                         <Col span={24}>
                             <Typography.Title>{contactRoleTitleMessage}</Typography.Title>
                         </Col>
+                        {canManageContactRoles && !get(contactRole, 'organization', null) && (<Col>
+                            <Row gutter={BIG_VERTICAL_GUTTER}>
+                                <Col span={24}>
+                                    <Alert
+                                        icon={<ExclamationCircleOutlined/>}
+                                        message={<strong>{ReadOnlyRoleWarningTitle}</strong>}
```

Comment:
```suggestion
                                        message={<Typography.Text strong>{ReadOnlyRoleWarningTitle}</Typography.Text>}
```

Reply:
No, in this case we will loose title color and have to set it manually.